woocommerce/woocommerce-android
# AI Product description: AI button in Aztec editor

Fixes #9337, a subtask of #9214.

This PR adds an AI description generation button to the Aztec editor.

[device-2023-07-01-121025.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/d3272a0f-ee51-4fdb-b362-f4a19a82a0b2)

**To test:**
1. Go to Products -> select a product
2. Tap on Description to open an editor
3. Notice there's a new button in the toolbar
4. Long click on the button and notice a toast with description is shown
5. Tap on the button
6. Notice an event is tracked: `🔵 Tracked: product_description_ai_button_tapped, Properties: {"source":"aztec_editor", ...}`
7. Notice the product name is displayed at the top
8. Proceed with description generation and tap on the `Apply` button
9. Notice the generated description is now in the editor

> **Note**
> Please merge #9340 first.
